collector/Dockerfile

```diff
@@ -4,3 +4,7 @@ RUN virtualenv venv
 COPY . /src
 RUN pip install -r /src/requirements.txt
 CMD python /src/main.py
+ENV API_USER=freqtrader
```

Comment:
These environment values should go before executing the script, otherwise they won't be set in our runtime - which is probably why we currently load them via `load_dotenv`, right?

Reply:
it should not matter. CMD is just the default command that is used as entrypoint. But i moved it up